src/api/libcellml/issue.h

```diff
@@ -188,21 +195,27 @@ class LIBCELLML_EXPORT Issue
         MATH_CN_UNITS,
         MATH_CN_BASE10,
         MATH_CN_FORMAT,
+        ENCAPSULATION_ATTRIBUTE,
         ENCAPSULATION_CHILD,
         COMPONENT_REF_COMPONENT,
         COMPONENT_REF_CHILD,
+        CONNECTION_ATTRIBUTE,
         CONNECTION_COMPONENT1,
         CONNECTION_COMPONENT2,
         CONNECTION_EXCLUDE_SELF,
         CONNECTION_UNIQUE,
         CONNECTION_CHILD,
+        MAP_VARIABLES_ATTRIBUTE,
         MAP_VARIABLES_VARIABLE1,
         MAP_VARIABLES_VARIABLE2,
         MAP_VARIABLES_UNIQUE,
         MAP_VARIABLES_AVAILABLE_INTERFACE,
 
         // Issues not present in the normative specification:
         MAP_VARIABLES_IDENTICAL_UNIT_REDUCTION,
+
+        // Placeholder for further references:
+        TODO
```

Comment:
not a fan of `TODO` - anything that is todo should not be going into the code :) otherwise why not `UNDEFINED`?

Reply:
It needs to be something separate from `UNDEFINED` ... maybe `NOT_DEFINED_YET`? It's just a placeholder until we've written the list of links to the docs and issue types.

Reply:
so that is just a reference that doesn't exist, right? so its undefined. once its defined it becomes defined...

Reply:
I interpret `undefined` to mean `I don't know what on earth you've done but it's way too freaky to have its own error type`.  What this enum is meant to be is `legit mistake, here's the docs, oh oops we haven't written that yet`.

Reply:
I've changed it to `NOT_SPECIFIED` in 54c49aa4